apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-resources/facility-resources.component.ts

```diff
@@ -109,4 +135,10 @@ export class FacilityResourcesComponent implements OnInit {
       }
     });
   }
+
+  serviceSelected(service: Service): void {
+    this.selectedService = service;
+    this.refreshTable();
+    this.cd.detectChanges();
```

Comment:
Don't think we need to detect changes here.

Reply:
Rarely a 'Expression has changed after it was checked' error was thrown when refresing the table. I couldn't quite figure out why but this fixed it